### ISSUE TITLE
Alt MMB navigation hotkey not working 

### ISSUE THREAD

**Issue by jlampel** [CGCookie/retopoflow#1142]
I'm not sure whether this is a bug or a feature request, but Blender's Alt MMB that focuses the view on whatever is under the mouse (which I only recently discovered) does not work in RetopoFlow. 

Possibly related to #1112 ?

**Comment by vxlcoder** [comment]
this issue is related to #1112, in that the action system isn't seeing this action yet, because i didn't implement the click handling correctly.  in other words, `view3d.view_center_pick` is already in list of navigation operators, but since that `KeyMapItem` is using click (`Alt+MiddleMouse+Click`) rather than press, the action system ignores that event.  changing the KMI to press instead works great!

I just pushed a huge revamp of the action system, but i'll need to do more work that's been on my long to-do before I can get this working.

**Comment by astradamus** [comment]
Any progress on this, or a workaround for similar functionality? I can't impress just how cumbersome it is to use the tool (or any 3d modeling tool) without the ability to freely place the camera pivot. When working on very convex shapes like hands/faces, it's incredibly difficult to move around quickly and is disastrous for my efficiency.

With this issue being 7 months old, I'm guessing that users in general have found some other workaround, and I'd love to know what that is. Otherwise, in every single way this is an amazing product and I'm extremely happy with my purchase :), just this one detail is really jamming me up :P

**Comment by jlampel** [comment]
At the moment, I center the pivot by selecting something and then focusing on the selection with the period key. It's not quite as convenient and does require drawing geometry in that area if there isn't any already, but it does the trick.

**Comment by astradamus** [comment]
Yep that works great. I guess I discovered alt-mmb early enough in my Blender journey that I never had to find this command. Had to rebind it as it was like ctrl-numpad-period which was a bit weird, but it works well enough! The only awkward thing is that Retopoflow seems to super-zoom in on the target when focusing (far more than happens outside the retopo editor), like putting the camera almost directly on top of it. Strange but not that big a deal.

Thanks for the tip!

**Comment by amarillosebas** [comment]
> this issue is related to #1112, in that the action system isn't seeing this action yet, because i didn't implement the click handling correctly. in other words, `view3d.view_center_pick` is already in list of navigation operators, but since that `KeyMapItem` is using click (`Alt+MiddleMouse+Click`) rather than press, the action system ignores that event. changing the KMI to press instead works great!
> 
> I just pushed a huge revamp of the action system, but i'll need to do more work that's been on my long to-do before I can get this working.

Is this still in the to-do list?

**Comment by DrakeGator** [comment]
If #1112 has now been fixed, what is keeping this from being fixed?

**Comment by dragos240** [comment]
This would be really helpful since I use this literally all the time in normal modes.

**Comment by DrakeGator** [comment]
🙏🙏🙏🙏🎉🎉🎉🎉🎉🥳🥳🥳 thank you!

**Comment by jlampel** [comment]
I can confirm that this is fixed. Thank you @jfranmatheu !
